jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Response.java

```diff
@@ -57,18 +57,10 @@ public interface Response extends Content.Writer
 
     HttpFields.Mutable getHeaders();
 
-    // TODO: change this to trailers(Supplier<HttpFields> supplier)
-    //  so that the method name is less confusing?
-    //  (it has a side effect, but looks like a normal getter).
-    HttpFields.Mutable getTrailers();
+    HttpFields.Mutable getOrCreateTrailers();
 
     @Override
-    void write(boolean last, Callback callback, ByteBuffer... content);
-
-    default void write(boolean last, Callback callback, String utf8Content)
-    {
-        write(last, callback, StandardCharsets.UTF_8.encode(utf8Content));
-    }
+    public void write(boolean last, Callback callback, ByteBuffer... buffers);
```

Comment:
Do we need this empty override?  At very least do:
```suggestion
    void write(boolean last, Callback callback, ByteBuffer... buffers);
```

Reply:
Done.